src/content/docs/fr/guides/authentication.mdx

```diff
@@ -188,6 +189,157 @@ const session = await getSession(Astro.request);
+};
+```
+
+Suivez le [guide Astro de Better Auth](https://www.better-auth.com/docs/integrations/astro) pour en savoir plus.
+
+### Utilisation
+
+Better Auth propose un assistant `createAuthClient` pour divers frameworks, dont Vanilla JS, React, Vue, Svelte et Solid.
+
+Par exemple, pour créer un client pour React, importez l'assistant depuis `'better-auth/react'` :
+
+
+
+<UIFrameworkTabs>
+  <Fragment slot="react">
+   ```ts title="src/lib/auth-client.ts"
+  import { createAuthClient } from 'better-auth/react';
+
+  export const authClient = createAuthClient();
+
+  export const { signIn, signOut } = authClient;
+  ```
+  </Fragment>
+  <Fragment slot="solid">
+  ```ts title="src/lib/auth-client.ts"
+  import { createAuthClient } from 'better-auth/solid';
+
+  export const authClient = createAuthClient();
+
+  export const { signIn, signOut } = authClient;
+  ```
+  </Fragment>
+  <Fragment slot="svelte">
+   ```ts title="src/lib/auth-client.ts"
+  import { createAuthClient } from 'better-auth/svelte';
+
+  export const authClient = createAuthClient();
+
+  export const { signIn, signOut } = authClient;
+  ```
+  </Fragment>
+  <Fragment slot="vue">
+  ```ts title="src/lib/auth-client.ts"
+  import { createAuthClient } from 'better-auth/vue';
+
+  export const authClient = createAuthClient();
+
+  export const { signIn, signOut } = authClient;
+  ```
+  </Fragment>
+</UIFrameworkTabs>
+
+Une fois votre client configuré, vous pouvez l'utiliser pour authentifier les utilisateurs dans vos composants Astro ou dans tout fichier spécifique au framework. L'exemple suivant ajoute la possibilité de se connecter ou de se déconnecter avec vos fonctions configurées `signIn()` et `signOut()`.
+
+
+```astro title="src/pages/index.astro"
+---
+import Layout from 'src/layouts/Base.astro';
+---
+<Layout>
+  <button id="login">Se connecter</button>
+  <button id="logout">Se déconnecter</button>
+
+  <script>
+    const { signIn, signOut } = await import("./lib/auth-client")
+    document.querySelector("#login").onclick = () => signIn.social({
+      provider: "github",
+      callbackURL: "/dashboard",
+    })
+    document.querySelector("#logout").onclick = () => signOut()
+  </script>
+</Layout>
+```
+
+Vous pouvez ensuite utiliser l'objet `auth` pour obtenir les données de session de l'utilisateur dans votre code côté serveur. L'exemple suivant personnalise le contenu de la page en affichant le nom d'un utilisateur authentifié :
+
+```astro title="src/pages/index.astro"
+---
+import { auth } from "../../../lib/auth"; // importe votre instance Better Auth
+ 
+const session = await auth.api.getSession({
+	headers: Astro.request.headers,
+});
+---
+
+<p>{session.user?.name}</p>
+```
+
+Vous pouvez également utiliser l'objet `auth` pour protéger vos routes à l'aide d'un middleware. L'exemple suivant vérifie si un utilisateur essayant d'accéder à une route de tableau de bord connectée est authentifié et le redirige vers la page d'accueil dans le cas contraire.
```

Comment:
I have some trouble with this sentence. Like "tableau de bord connectée" mean first of all "connected to internet", but the real purpose of this sentence is "logged-in".
I have no idea of how we can formulate this without being to boring (Réberbatif).
`L'exemple suivant vérifie si, un utilisateur essayant d'accéder à une route de tableau de bord nécessitant d'être connecté, est authentifié et le redirige vers la page d'accueil dans le cas contraire.`
But for me it's too much.

Reply:
You're right, maybe it wasn't the right term! We could use `protégé`: either `accéder à une route de tableau de bord protégé` or `accéder à une route protégée de tableau de bord`. Does that seem clearer to you?

Reply:
Ah I didn't see your edit. I must have opened the page before... So we agree!